tests/self/se_time.c

```diff
@@ -157,3 +157,26 @@ void selftest_delay(void)
 }
 
 REGISTER_SELF_TEST(delay, se_manual, &selftest_delay)
+
+void selftest_latency(void)
+{
+   struct datetime d;
+   const int iters = 10000;
```

Comment:
10k iters are not enough to stabilize the result. I tried it myself. The latency varies between 3k and 4k cycles.
That's not OK. You have to increase the iterations until the results vary a very little. A perf test like this lasts typically a few seconds. This runs immediately and doesn't produce stable results.

Reply:
With 200k iters and 20 second test time, the latency varies < 2k cycles. Is this acceptable?

Reply:
@rviu Sorry, what do you mean with "the latency varies < 2k cycles" ? I understand that it falls below 2k, but how much it varies? A good-enough measurement varies by 2%. If it varies too much with too many iterations, we have to understand why. It's possible that it varies because of other threads, in particular the thread that reads the time intensively I talked about. So, I suggest putting: `disable_preemption()` and `enable_preemption()` around the loop and see if you can get more stable results with fewer iterations. The criteria is 2% variance, with fewer iterations as possible, of course.

Reply:
> The latency varies between 3k and 4k cycles.

Oh, I just understood what you meant by this. I thought you were referring to how much the difference b/w maximum and minimum observed latency was for you.

It's completely different for me. For me the latency varies between 310,000 and 312,000. This is what I meant when saying < 2k cycles.

I'm running the self-test inside Tilck by running `selftest latency`.

Reply:
@rviu Aha, got it. Yeah, the actual number varies a lot depending on the environment. E.g. if you use KVM vs. pure software virtualization, it's usually a big difference.

20 seconds are a bit too much. Can you disable the preemption and bring down the number of iterations so that results are stable both with KVM and without it? It should be possible to get stable results in a few seconds.

Reply:
@vvaltchev Now with 50k iters, it takes about 6 seconds using kvm and almost instantly with non kvm well within the variance criteria

Reply:
That's good, thank you!